ci_config.json

```diff
@@ -31,6 +31,29 @@
       "tzdata"
     ]
   },
+  "arrow-cpp": {
+    "_comment": "Windows symbol visibility broken; no upstream Windows CI to fix yet",
+    "build_on": {
+      "msys2": false,
+      "windows": false
+    },
```

Comment:
These directives declare that the project actively doesn't support Windows/MSYS.  They should be removed given https://github.com/mesonbuild/wrapdb/pull/2099#issuecomment-3151340574.